ProcessLib/SmallDeformation/Tests.cmake

```diff
@@ -102,6 +103,21 @@ AddTest(
     ../../ring_plane_strain_pcs_0_ts_1_t_1.000000.vtu ring_plane_strain_pcs_0_ts_1_t_1.000000.vtu displacement displacement 1e-16 0
     ../../ring_plane_strain_pcs_0_ts_1_t_1.000000.vtu ring_plane_strain_pcs_0_ts_1_t_1.000000.vtu sigma sigma 1e-15 0
 )
+
+#slope stability test with MFront Mohr Coulomb implementation
+#Todo Tengfei ... input correct reference files
+AddTest(
```

Comment:
Either `AddTest` or `OgsTest`, not both. For `OgsTest` version you need to define a [`test_definition`](https://doxygen.opengeosys.org/ogs_file_param__prj__test_definition.html) section in the project file, like it is done in other `OgsTest`-teste files.
In the above `OgsTest` the path is wrong.

Reply:
You could change the <t_end> to 5 and test the benchmark. I have to change the end time to test it because the program does not stop once it reaches dt_min and still cannot converge.

Reply:
I will change it and upload the project file